### PR TITLE
Added sprite and palettes for Teal Mask items and updated install instructions

### PR DESCRIPTION
**Pull request by hedara90**
## Description
Added a single sprite for the all Mochi items introduced in the Teal Mask with different palettes for each option.
Added sprite for Syrupy Apple.
Added sprites for Sinistea teacups.
Added sprites for Ogerpon masks.
Added sprite for Glimmering Charm.
Updated INSTALL.md to account for the switch to build the MODERN option by default which requires more packages on Linux based environments.

## Images
allTealItems

## **Discord contact info**
hedara#5073

### CODE REVIEW

**Comment by AsparagusEduardo** [comment]
> Waiting for [#3530 ](https://github.com/rh-hideout/pokeemerald-expansion/pull/3530) to be merged. CI will fail until then.

Can you temporarely apply the changes in that PR to make sure that it solves the issue?

**Comment by hedara90** [comment]
Applied the fix.

File: graphics/items/icons/health_mochi.png
Comment (AsparagusEduardo) [inline graphics/items/icons/health_mochi.png:1]: Rename to just `mochi.png`

File: INSTALL.md
Comment (AsparagusEduardo) [inline INSTALL.md:1]: Can you go into detail why update these instructions?

**Review by AsparagusEduardo — CHANGES_REQUESTED** [review]

**Comment by hedara90** [comment]
> > Waiting for [#3530 ](https://github.com/rh-hideout/pokeemerald-expansion/pull/3530) to be merged. CI will fail until then.
> 
> Can you temporarely apply the changes in that PR to make sure that it solves the issue?

Those changes were supposed to be in a different branch I had, I don't know how I messed up that.
But the changes are due the switch to using MODERN=1 as the default build option. On Linux systems the project won't compile without the packages arm-none-eabi-gcc and arm-none-eabi-newlib on Arch and their equivalent on Ubuntu.
So I was updating the installation instructions to match the new requirements.
2023-11-05T11:04:31
2023-11-05T10:56:23

File: INSTALL.md
Comment (hedara90) [reply INSTALL.md:1]: Apparently didn't write the reply in the correct box [here it is](https://github.com/rh-hideout/pokeemerald-expansion/pull/3527#issuecomment-1794362492)

Comment (AsparagusEduardo) [reply INSTALL.md:1]: Understood.

**Comment by hedara90** [comment]
I finished the rest of the items (excluding glimmering charm), I can add them too. (and rename the PR, since it's gone beyond the scope of initial title)

**Comment by AsparagusEduardo** [comment]
> I finished the rest of the items (excluding glimmering charm), I can add them too. (and rename the PR, since it's gone beyond the scope of initial title)

Sure, it should still be a small-enough to not disrupt the previously-reviewed stuff :)

File: graphics/items/icons/health_mochi.png
Comment (hedara90) [reply graphics/items/icons/health_mochi.png:1]: Renamed the file.
And added the rest of the items

**Comment by AsparagusEduardo** [comment]
Please update title and description (with images) to reflect the newer changes.

**Review by AsparagusEduardo — APPROVED** [review]

**Pull request merged**